chatroom-socket-server/server.js

```diff
@@ -0,0 +1,43 @@
+const port = 8000;
+const httpServer = require("http").createServer((req, res) => {}).listen(port);
+const io = require("socket.io")(httpServer, {
+  cors: {
+    origin: "http://localhost:3000",
```

Comment:
So this is opening the socket on port 3000 in order to send requests to the http server on port 8000?

Reply:
Or this is simply saying connections from port 3000 are allowed to send requests to the 8000 server